README.md

```diff
@@ -203,10 +203,24 @@ These files are generated via the `i18n:update` NPM command, and like translatio
 
 ### How to add a new plugin
 1. Identify the i18n domain, which usually matches the plugin's name (i.e. `jetpack`).
-2. Identify the path to the plugin source code (i.e. `./jetpack/projects/plugins/jetpack/extensions`).
-3. Append the plugin's name to the arguments of `i18n:check-cache` NPM command.
-4. Append the plugin's name and source code path to the arguments of `i18n:update` NPM command.
-5. Add the i18n domain of the plugin and the callback for getting translation to the [editor initialization](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/index.js).
+2. Identify the GlotPress project slug (i.e. `wp-plugins/jetpack` for URL `https://translate.wordpress.org/projects/wp-plugins/jetpack/`)
+3. Identify the path to the plugin's source code (i.e. `./jetpack/projects/plugins/jetpack/extensions`).
+4. Append the plugin's name, GlotPress project slug, and plugin's source code to the arguments of `i18n:update` NPM command.
+5. Append the plugin's name, GlotPress project slug, and plugin's source code to the arguments of `i18n:update:test` NPM command.
+6. Append the plugin's name and GlotPress project slug to the arguments of `i18n:check-cache` NPM command.
```

Comment:
Given how repetitive these steps are, it may be clearer to combine them. WDYT? 

```suggestion
4. Append the plugin's name, GlotPress project slug, and plugin's source code to the arguments of `i18n:update` and `i18n:update:test` NPM commands.
5. Append the plugin's name and GlotPress project slug to the arguments of `i18n:check-cache` NPM command.
```

Reply:
Yeah, good idea 🙇 ! I'll apply your suggestion.